db/data/my.bookshop-Genres.csv

```diff
@@ -1,31 +1,34 @@
-ID;parent_ID;name
-100;;Fiction
-101;100;Action
-102;100;Adventure
-103;100;Drama
-105;100;Poetry
-106;100;Science Fiction
-107;106;Utopian and Dystopian
-108;107;Dystopia
-109;108;Cyberpunk
-110;109;Steampunk
-104;100;Fantasy
-111;104;Epic fantasy
-112;104;High fantasy
-113;100;Graphic Novel
-115;100;Short Story
-116;100;Romance
-117;100;Mystery
-123;117;Thriller
-124;117;Suspense
-118;100;Horror
-119;100;Historical Fiction
-120;100;Contemporary Fiction
-121;100;Magical Realism
-122;100;Literary Fiction
-128;100;Fairy Tale
-200;;Non-Fiction
-201;200;Biography
-202;201;Autobiography
-203;200;Essay
-204;200;Speech
+ID;parent_ID;name;siblingRank
```

Comment:
Its always a little confusing when debugging, when so many UUIDs look so similar. Maybe actually generate random UUIDs for all of these genres.